doc/guides/dts/intro.rst

```diff
@@ -281,6 +282,22 @@ reg
     property can be seen as a more detailed view of the addressable resources
     within a device than its unit address.
 
+status
+    A string which describes whether the node is enabled.
+
+    The devicetree specification allows this property to have values
+    ``"okay"``, ``"disabled"``, ``"reserved"``, ``"fail"``, and ``"fail-sss"``.
+    Only the values ``"okay"`` and ``"disabled"`` are currently relevant to
+    Zephyr; use of other values currently results in undefined behavior.
+
+    A node is considered enabled if its status property is either ``"okay"`` or
+    not defined. Nodes with status ``"disabled"`` are explicitly disabled. (For
```

Comment:
I wonder if 'not defined' isn't quite clear based on the previous paragraph.  Maybe 'not defined (ie does not exist in the dts)'?

Reply:
or "not provided" or "missing" or "absent".  I see "missing" used at least once in the existin documentation:

> (howtos.rst) node's ``compatible`` property is missing, or its value has no matching

Reply:
changed both to 'not defined'